### DISCUSSION TITLE
forward email tickets to an external email address directly

### DISCUSSION THREAD

**Discussion by abdullah-s2** [chatwoot/chatwoot#9667] [Feature Requests]
We need the ability to forward email tickets to an external email address directly from within the system. This feature would greatly simplify sharing ticket details with external parties without the need for manually copying and pasting content or using complex alternative solutions.

I would greatly appreciate it if you could consider this request and add it to the upcoming roadmap.

Thank you for your continuous support.

**Comment by DaalderConcepts** [comment]
Agreed, one of the most annoying things in Chatwoot! Now we need to open webmail first to forward. Not very handy.

**Comment by sojan-official** [comment]
Thanks for raising this. Email forwarding from Chatwoot is already tracked in chatwoot/chatwoot#12691, so I am closing this discussion as a duplicate to keep follow-up in one place.
